package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "eddiehub",
-  "version": "0.15.5",
+  "version": "0.15.6",
```

Please do not bump the version

```suggestion
  "version": "0.15.5",
```